packages/blockchain-wallet-v4-frontend/src/scenes/Settings/General/LinkedBanks/index.tsx

```diff
@@ -1,102 +1,36 @@
-    const ACHProvider = plaidEnabled ? 'ADD_BANK_PLAID_MODAL' : 'ADD_BANK_YODLEE_MODAL'
-    this.props.brokerageActions.showModal({
-      modalType: tradingCurrency === 'USD' ? ACHProvider : 'ADD_BANK_YAPILY_MODAL',
-      origin: BrokerageModalOriginType.ADD_BANK_SETTINGS
-    })
-
-    this.props.brokerageActions.setAddBankStep({
-      addBankStep: AddBankStepType.ADD_BANK
-    })
-  }
-
-  handleShowBankClick = (account: BankTransferAccountType) => {
-    this.props.brokerageActions.showModal({
-      modalType: 'BANK_DETAILS_MODAL',
-      origin: BrokerageModalOriginType.BANK
-    })
-    this.props.brokerageActions.setBankDetails({
-      account
-    })
-  }
-
-  handleDeleteBank = (account: BankTransferAccountType) => {
-    this.props.brokerageActions.showModal({
-      modalType: 'REMOVE_BANK_MODAL',
-      origin: BrokerageModalOriginType.BANK
-    })
-    this.props.brokerageActions.setBankDetails({
-      account
-    })
-  }
-
-  render() {
-    return this.props.data.cata({
-      Failure: () => null,
-      Loading: () => <Loading />,
-      NotAsked: () => <Loading />,
-      Success: (val) => (
-        <Success
-          {...this.props}
-          {...val}
-          handleBankClick={this.handleBankClick}
-          handleShowBankClick={this.handleShowBankClick}
-          handleDeleteBank={this.handleDeleteBank}
-        />
-      )
-    })
-  }
+type DataType = {
+  bankAccounts: BankTransferAccountType[]
+  paymentMethods: BSPaymentMethodsType
 }
 
-const mapStateToProps = (state: RootState): LinkStatePropsType => ({
-  data: getData(state),
-  plaidEnabled: selectors.core.walletOptions.getAddPlaidPaymentProvider(state).getOrElse(false),
-  tradingCurrency: selectors.modules.profile.getTradingCurrency(state).getOrElse('USD'),
-  userData: selectors.modules.profile.getUserData(state).getOrElse({} as UserDataType)
-})
+const LinkedBanks = () => {
+  const { data, hasError, isLoading, isNotAsked } = useRemote(getData)
+  const tradingCurrency = useSelector(getTradingCurrency).getOrElse('USD')
 
-const mapDispatchToProps = (dispatch: Dispatch) => ({
-  brokerageActions: bindActionCreators(actions.components.brokerage, dispatch),
-  buySellActions: bindActionCreators(actions.components.buySell, dispatch),
-  custodialActions: bindActionCreators(actions.custodial, dispatch),
-  withdrawActions: bindActionCreators(actions.components.withdraw, dispatch)
-})
+  const dispatch = useDispatch()
 
-const connector = connect(mapStateToProps, mapDispatchToProps)
+  useEffect(() => {
+    dispatch(brokerage.fetchBankTransferAccounts())
+    dispatch(buySell.fetchPaymentMethods(tradingCurrency))
+  }, [])
 
-type LinkStatePropsType = {
-  data: RemoteDataType<string, SuccessStateType>
-  plaidEnabled: unknown | boolean
-  tradingCurrency: WalletFiatType
-  userData: UserDataType
+  if (hasError) return <></>
+  if (isLoading || isNotAsked) return <Loading />
+  if (!data) return <></>
```

Comment:
we could combine line 30 and 32. Also, we can just return `null` instead of `<></>`